sbi/inference/base.py

```diff
@@ -152,6 +156,108 @@ def __init__(
             median_observation_distances=[], epochs=[], best_validation_log_probs=[],
         )
 
+    def provide_presimulated(
+        self, theta: Tensor, x: Tensor, from_round: int = 0
```

Comment:
use the same argument name as in `_append_to_round_bank`, i.e., `round_`, or maybe even `round_idx`?

Reply:
Calling all of them `from_round`.